src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ViewCellRenderer.cs

```diff
@@ -244,6 +244,12 @@ public void Update(ViewCell cell)
 
 				var platformView = _viewCell.View.ToPlatform(Element.FindMauiContext());
 				_viewHandler = (IPlatformViewHandler)_viewCell.View.Handler;
+
+				if (platformView.Parent != null && platformView.Parent is ViewGroup parentViewGroup)
+				{
+					parentViewGroup.RemoveView(platformView);
```

Comment:
The `RemoveFromParent` extension method can be used here.

`platformView.RemoveFromParent();`